src/edge/recombine.py

```diff
@@ -146,23 +146,35 @@ def get_cassette_inherited_annotations(genome, cassette, fragment_id, region_sta
     return inherited_annotations
 
 
-def get_cassette_new_annotations(cassette):
+def get_cassette_new_annotations(cassette, new_annotations):
     # detect ORF on cassette
 
     annotations = []
     for orf in detect_orfs(cassette):
         annotations.append(dict(base_first=orf['start'], base_last=orf['end'],
                                 feature_name=orf['name'],
                                 feature_type='ORF',
-                                feature_strand=orf['strand']))
+                                feature_strand=orf['strand'],
+                                feature_qualifiers=None))
+
+    if new_annotations:
+        for annotation in new_annotations:
+            annotations.append(
+                dict(base_first=annotation['base_first'], base_last=annotation['base_last'],
+                     feature_name=annotation['name'],
+                     feature_type=annotation['type'],
+                     feature_strand=annotation['strand'],
+                     feature_qualifiers=annotation['qualifiers']
```

Comment:
Can we update here to use annotation.get('qualifiers') as well?

Reply:
Done